src/main/java/fr/techad/edc/client/internal/io/HttpReaderImpl.java

```diff
@@ -291,6 +278,42 @@ private DocumentationItemType getType(JsonObject jsonObject) {
 
     }
 
+    @Override
+    public Table<String, String, String> readi18nContent(String languageCode, String i18n_label) throws IOException, InvalidUrlException {
```

Comment:
Encapsulate this Table in a class to declare the contract.
In fact, it is difficult to use it because we don't know if the first argument is the lang, the key, ...
If you encapsulate this Table in a class, you will implement getLabel(String lang, String key) and the contract will clear...

More, you have to read all langagueCodes like before.
In java, i18n_label is not relevant, i18nLabel is better.